jfrog-applications/frogbot/scan-repositories.md

```diff
@@ -29,4 +29,8 @@ The following alert types are supported:
 
 **4. Static Application Security Testing (Sast) vulnerabilities**
 
-![](../.gitbook/assets/github-code-scanning-sast-content.png)
\ No newline at end of file
+![](../.gitbook/assets/github-code-scanning-sast-content.png)
+
+**5. Validate Dependency Licenses**
+
```

Comment:
Let's add the following description for this section:

When Frogbot scans the repository periodically, it checks the licenses of any project dependencies. If Frogbot identifies licenses that are not listed in a predefined set of approved licenses, it adds an alert. The list of allowed licenses is set up as a variable within the Frogbot workflow.

Also, for the same section under the "Scan Pull Requests" section, let's replace the existing description with the following:

When Frogbot scans newly opened pull requests, it checks the licenses of any new direct project dependencies introduced by the pull request. If Frogbot identifies licenses that are not listed in a predefined set of approved licenses, it appends a comment to the pull request providing this information. The list of allowed licenses is set up as a variable within the Frogbot workflow.